examples/graphql-yoga/src/server.js

```diff
@@ -52,8 +52,12 @@ const server = new GraphQLServer({
 
     // Using the same EXPECTED_OPTIONS_KEY, store the DataLoader context
     // in the global request context
+    resolver.contextToOptions = {
+      dataloaderContext: [EXPECTED_OPTIONS_KEY]
+    };
+
     return {
-      [EXPECTED_OPTIONS_KEY]: dataloaderContext,
+      dataloaderContext,
```

Comment:
old way would have worked just fine as long as context maps `EXPECTED_OPTIONS_KEY` to `EXPECTED_OPTIONS_KEY`

Reply:
Yeap, you're right, just updating the resolvers I got the correct result.

Reply:
I removed this change, since it was not necessary.